apps/site/src/lib/admin/useHackerApplicants.ts

```diff
@@ -0,0 +1,34 @@
+import axios from "axios";
+import useSWR from "swr";
+
+import { Decision, Status } from "./useApplicant";
```

Comment:
Note: these are being moved in #519, depending on whichever is merged first, will need to update the latter.

Reply:
#519 merged first, so this PR will need to be updated